client/src/templates/Introduction/components/block.tsx

```diff
@@ -238,39 +238,54 @@ export class Block extends Component<BlockProps> {
       </>
     );
 
+    const courseCompletionStatus = () => {
+      if (completedCount === 0) {
+        return 'not started';
+      }
+      if (completedCount === challengesWithCompleted.length) {
+        return 'completed';
+      }
+      return `${completedCount} of ${challengesWithCompleted.length} steps completed`;
+    };
```

Comment:
We should probably translations for these. Other than that LGMT

Reply:
@ahmadabdolsaheb I just noticed that we also have "M of N steps completed" that needs to be translated too. How would you handle that, since the M and N will change?

Reply:
@ahmadabdolsaheb I just noticed that we also have "M of N steps completed" that needs to be translated too. How would you handle that, since the M and N will change?

Reply:
something like this should do it for now. https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/i18n/locales/english/translations.json#L526

@moT01, some of the languages don't use the english numbers. What do you think would be the best course of action moving forward.

Reply:
@ahmadabdolsaheb @moT01 Would it complicate things less if instead of saying "M of N steps completed" we use a percentage instead? Such as "N% completed". Visually we are showing a percentage, so this would match the visual a little better.

Reply:
Yes, let's do percentages.

Reply:
Done.